yarn.lock

```diff
@@ -2,395 +2,3 @@
 # yarn lockfile v1
```

Comment:
Can we delete this file entirely? We shouldn't have a yarn.lock file at this level of the project 🙂

Reply:
Agree, I think it will come back anyway as soon as we migrate to yarn workspaces but let's remove it for now!